enterprise-ws-spec/src/main/asciidoc/ServerProgrammingModel.adoc

```diff
@@ -86,7 +83,7 @@ programming model artifacts include:
 WSDL document – Although not strictly a programming model artifact, the
 WSDL document provides a canonical description of a Web service that may
 be published to third parties. A WSDL document and the Service Endpoint
-Interface are related by the JAX-RPC/JAX-WS WSDL<->Java mapping rules.
+Interface are related by the Jakarta XML Web Services WSDL<->Java mapping rules.
```

Comment:
this "<->" looks strange in generated html, can sth be done with it?

Reply:
How about WSDL<=>?

Reply:
the same effect - left character is kept as is, remaining two are translated to something someone thinks looks better :-/ let's keep it as it is for now

Reply:
OK, I see WSDL<=>Java used in JAXS 2.2 spec.